python/lsst/pipe/tasks/ingest.py

```diff
@@ -307,20 +304,32 @@ def createTable(self, conn, table=None):
         @param conn    Database connection
         @param table   Name of table to create in database
         """
+        cursor = conn.cursor()
         if table is None:
             table = self.config.table
+        cmd = "SELECT name FROM sqlite_master WHERE type='table' AND name='%s'" % table
```

Comment:
I strongly suggest you leave `%s` in the string and pass the argument separately to `cursor.execute` -- both here and in the other simple sql commands below (it may be too much work for the more complicated command). This allows `execute` to sanitize the inputs for you, to prevent SQL injection. For example:
```
cmd = "SELECT name FROM sqlite_master WHERE type='table' AND name='%s'"
cursor.execute(cmd, table)
```
In cases where you don't think it's worth the effort please consider using f strings, as they are easier to read.